src/transformers/utils/quantization_config.py

```diff
@@ -49,7 +53,7 @@ class BitsAndBytesConfig:
             These outliers are often in the interval [-60, -6] or [6, 60]. Int8 quantization works well for values of
             magnitude ~5, but beyond that, there is a significant performance penalty. A good default threshold is 6,
             but a lower threshold might be needed for more unstable models (small models, fine-tuning).
-        llm_int8_skip_modules (`List[str]`, *optional*):
+        bnb_kbit_skip_modules (`List[str]`, *optional*):
```

Comment:
We can't rename an argument here without care for backward compatibility. There are now quantization configs on the Hub with this argument set.